tests/Unit/SDK/Resource/ResourceTest.php

```diff
@@ -293,19 +294,28 @@ public function test_resource_service_name_default(): void
         $this->assertEquals('unknown_service', $resource->getAttributes()->get('service.name'));
     }
 
+    /**
+     * @group compliance
```

Comment:
Should these be marked as `@group trace-compliance`?

Reply:
@bobstrecansky I was actually not sure about this, because this could be both a part of `@group metric-compliance` and `@group trace-compliance` from what I understand.

Reply:
happy with "compliance" because it's not specific to any signal